wrappers/python/tests/pool/test_delete_pool_ledger_config.py

```diff
@@ -1,21 +1,29 @@
-from tests.utils import pool
-from indy.pool import delete_pool_ledger_config
+import json
+
+from tests.utils import pool as pool_utils
+from indy import pool
 from indy.error import ErrorCode, IndyError
 
 import pytest
 
 
 @pytest.mark.asyncio
 async def test_delete_pool_ledger_config_works(cleanup_storage):
-    await pool.create_pool_ledger_config("pool_1")
-    await delete_pool_ledger_config("pool_1")
+    await pool.create_pool_ledger_config(
+        "pool_1",
+        json.dumps({
+            "genesis_txn": str(pool_utils.create_genesis_txn_file_for_test_pool("pool_1"))
+        }))
+
+    await pool.delete_pool_ledger_config("pool_1")
```

Comment:
See my comment about create_pool_ledger_config. We have exactly same situation here.